addons/website_forum/wizard/forum_post_state_change.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
```

Comment:
Know I understand, I completely made abstraction that portal users don't have access to the backend. It's why my security rules are wrong. 

And for the spec, creating a wizzard to specify the offensive or closed reason seems the best option in my mind (maybe I'm wrong).

Reply:
Yes, we need a (only 1) wizard in case we keep it. 
But imo, it is overkill and add code / security rule / ... It is why we should discuss about it tommorow with sbu to know if we really want to keep it or not.